css/color-scheme.css

```diff
@@ -130,6 +73,20 @@ body {
     font-size: 110%;
 }
 
+/* header */
+
+.header-base img {
+    display: block;
+    max-width: 500px;
```

Comment:
use rem for sizes greater than 16px

Reply:
This was to display the container, so using pixels helps me visualize better.